src/graphic/Fast3D/gfx_pc.cpp

```diff
@@ -1826,6 +1830,10 @@ static void gfx_sp_movemem_f3d(uint8_t index, uint8_t offset, const void* data)
         case G_MV_VIEWPORT:
             gfx_calc_and_set_viewport((const Vp_t*)data);
             break;
+        case G_MV_LOOKATY:
+        case G_MV_LOOKATX:
```

Comment:
`error: ‘G_MV_LOOKATX’ was not declared in this scope; did you mean ‘G_MVO_LOOKATX’?`